src/commands/push-to-docker.yaml

```diff
@@ -0,0 +1,14 @@
+parameters:
+  image:
+    type: "string"
+  password:
+    type: "string"
+  registry:
+    type: "string"
+  username:
+    type: "string"
+steps:
+  - run: echo -n "<<parameters.image>>:$(architect project version)" > docker_image_name
+  - run: docker login --username "<<parameters.username>>" --password "<<parameters.password>>" "<<parameters.registry>>"
+  - run: docker build -t "$(cat docker_image_name)" .
+  - run: docker push "$(cat docker_image_name)"
```

Comment:
I would strongly prefer to write it in this syntax:

```
  - run: |
      echo -n "<<parameters.image>>:$(architect project version)" > docker_image_name
```

Otherwise we can get into various yaml traps.

Reply:
When there is a need for multiline yes, but this fit perfectly in single lines here and I would prefer to keep it this way for readability.

Reply:
Updated this to multilines as there are weird cases where this fails (when using pipes or so?).